pkg/subctl/datafile/backup.go

```diff
@@ -0,0 +1,17 @@
+package datafile
+
+import (
+	"os"
+	"strings"
+	"time"
+)
+
+func BackupIfExists(filename string) (string, error) {
+	if _, err := os.Stat(filename); os.IsNotExist(err) {
+		return "", nil
+	}
+	now := time.Now()
```

Comment:
Nit: seems cleaner just to embed this in the next line vs having a dedicated var